exe/wallet/Main.hs

```diff
@@ -153,20 +172,26 @@ Usage:
   cardano-wallet --version
 
 Options:
+  --address-pool-gap <INT>    number of unused consecutive addresses to keep track of [default: 20]
+  --bridge-port <INT>         port used for communicating with the http-bridge [default: 8080]
+  --database <FILE>           use this file for storing wallet state
+  --network <STRING>          testnet or mainnet [default: testnet]
+  --payment <PAYMENT>         address to send to and amount to send separated by @: '<amount>@<address>'
   --port <INT>                port used for serving the wallet API [default: 8090]
   --random-port               serve wallet API on any available port (conflicts with --port)
-  --bridge-port <INT>         port used for communicating with the http-bridge [default: 8080]
-  --address-pool-gap <INT>    number of unused consecutive addresses to keep track of [default: 20]
   --size <INT>                number of mnemonic words to generate [default: 15]
-  --payment <PAYMENT>         address to send to and amount to send separated by @: '<amount>@<address>'
-  --network <STRING>          testnet or mainnet [default: testnet]
-  --database <FILE>           use this file for storing wallet state
   --state <STRING>            address state: either used or unused
+  --state-dir <DIR>           write wallet state (blockchain and database) to this directory
 
 Examples:
+  # Launch and monitor a wallet backend and its associated node
+  cardano-wallet launch --network mainnet --random-port --state-dir .state-dir
+
+  # Start only a wallet server and connect to an already existing backend service
+  cardano-wallet serve --bridge-port 8080
```

Comment:
It might be a bit confusing if someone does not have a context (i.e. is `wallet backend` different than `wallet server`) I think I'd rephrase this so it is more clear, e.g.:

```
  # Launch and monitor a wallet server and its associated backend service
  cardano-wallet launch --network mainnet --random-port --state-dir .state-dir

  # Start only a wallet server and connect to an already existing backend service
  cardano-wallet serve --bridge-port 8080
```